docs/groovy/reference/table-operations/select/update-view.md

```diff
@@ -18,6 +18,7 @@ When using `updateView`, the new columns are not stored in memory. Rather, a for
 
 > [!CAUTION]
 > When using [`view`](./view.md) or `updateView`, non-deterministic methods (e.g., random numbers, current time, or mutable structures) produce _unstable_ results. Downstream operations on these results produce _undefined_ behavior. Non-deterministic methods should use [`select`](./select.md) or [`update`](./update.md) instead.
+> Concurrency control (serial marking and barriers) cannot be applied to [`view`](./view.md) or [`updateView`](./update-view.md). These operations compute results on demand and cannot enforce ordering constraints. If you need serial evaluation or barriers, use [`select`](./select.md) or [`update`](./update.md) instead. See [Parallelizing queries](../../../conceptual/query-engine/parallelization.md#serialization) for more information.
```

Comment:
Should the link also appear at the bottom of the document?  I could go either way.  Should there also be a link to the parallelization quick start?